connectors/salesforce-upsert-contact-connector/src/main/resources/camel-connector-schema.json

```diff
@@ -18,10 +18,10 @@
   },
   "componentProperties": {
     "loginUrl": { "kind": "property", "displayName": "Login Url", "group": "security", "label": "common,security", "required": true, "type": "string", "javaType": "java.lang.String", "deprecated": false, "secret": false, "defaultValue": "https://login.salesforce.com", "description": "URL of the Salesforce instance by default set to https://login.salesforce.com" },
-    "clientId": { "kind": "property", "displayName": "Client Id", "group": "security", "label": "common,security", "required": true, "type": "string", "javaType": "java.lang.String", "deprecated": false, "secret": false, "description": "OAuth Consumer Key of the connected app configured in the Salesforce instance setup. Typically a connected app needs to be configured but one can be provided by installing a package." },
+    "clientId": { "kind": "property", "displayName": "Client Id", "group": "security", "label": "common,security", "required": true, "type": "string", "javaType": "java.lang.String", "deprecated": false, "secret": true, "description": "OAuth Consumer Key of the connected app configured in the Salesforce instance setup. Typically a connected app needs to be configured but one can be provided by installing a package." },
```

Comment:
I think we wanted `clientId` to be non-secret see #55

Reply:
hmm the only reason that would change is bc I ran the build. I'm working in the sql-stored-connector

Reply:
Yeah, I saw this too with newer version of Camel, I guess the camel-maven-package/connector plugins might generate code differently between versions; and quite possibly that I've changed that from `true` to `false` by hand. I would keep it out of this PR if possible and regenerate in the next PR (I'll create one for upgrading Camel version).